server/controllers/roundAnalysisCtrl.js

```diff
@@ -193,6 +193,32 @@ async function getUsedRounds(req, res, next) {
   }
 }
 
+async function getCanvasAnalysisListByCourseCode(req, res, next) {
+  const courseCode = req.params.courseCode.toUpperCase()
+  let dbResponse
+  try {
+    dbResponse = await db.fetchAllCanvasRoundAnalysesByCourseCode(courseCode)
+    log.debug('Successfully got all analyses for', { courseCode, dbResponse })
+    res.json(dbResponse)
+  } catch (err) {
+    log.error('Error in getCanvasAnalysisListByCourseCode', { error: err })
+    next(err)
+  }
+}
+
+async function getKursinfoadminAnalysisListByCourseCode(req, res, next) {
```

Comment:
Kursinfoadmin i the place where the admin tool lives right now, which is why this name was chosen. But Iäll think it through one more time when I fix the other comments :)

Reply:
And add documentation for it ofc.

Reply:
Yeah you are right! 

For me it wasn't clear at first that "Kursinfoadmin" referred to the whole group of admin apps. I initially thought of "kursinfo-admin-web" when I read "kursinfoadmin", but I realise now that "kursinfoadmin" is in the url for the other admin repos/apps as well (so that includes kursutveckling-admin-web).

Reply:
That is an easy fix! Thought of it but then thought it would feel risky to assume all other analyses weren't from canvas. But now I agree it may be the easiest and most clear solution :) We'll go for that!

Reply:
Also updated the Swagger docs and README with more info, for traceability.